Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/App.config

```diff
@@ -32,4 +33,16 @@
       </dependentAssembly>
     </assemblyBinding>
   </runtime>
+  <system.web>
+    <membership defaultProvider="ClientAuthenticationMembershipProvider">
```

Comment:
removing them as they are not required. Thanks. Most of changes were done to enforce some version of .net or some other things as it was not working first. But looks like we don't need it anymore.